modules/stanford_profile_styles/lib/scss/component/paragraph/card/_card.scss

```diff
@@ -0,0 +1,17 @@
+@charset 'UTF-8';
+
+.su-card {
+  .su-card__contents {
+    > h3 {
```

Comment:
why `>` ? this eliminates the possibility for structure like this: 
```
<div class="su-card__contents">
<div>
<h3>
```

Reply:
That's how the card h2's are currently styled in [decanter](https://github.com/SU-SWS/decanter/blob/v6/core/src/scss/utilities/mixins/card/_card.scss), but I see your point. I removed the `>` and updated the h2 as well so it won't have that issue either.